src/mono/wasi/wasi.proj

```diff
@@ -6,8 +6,9 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <ICULibDir Condition="'$(MonoWasmThreads)' != 'true'">$([MSBuild]::NormalizeDirectory('$(PkgMicrosoft_NETCore_Runtime_ICU_Transport)', 'runtimes', 'wasi-wasm', 'native', 'lib'))</ICULibDir>
-    <ICULibDir Condition="'$(MonoWasmThreads)' == 'true'">$([MSBuild]::NormalizeDirectory('$(PkgMicrosoft_NETCore_Runtime_ICU_Transport)', 'runtimes', 'wasi-wasm-threads', 'native', 'lib'))</ICULibDir>
+    <!-- TODOWASI: icu support -->
```

Comment:
Could we enable this again before merge or this breaks stuff ? 
cc @mkhamoyan

Reply:
I'll try.

Reply:
With this uncommented, a test build fails with:
```
wasm-ld : error : /workspaces/runtime/artifacts/bin/microsoft.netcore.app.runtime.wasi-wasm/Release/runtimes/wasi-wasm/native/libicudata.a: archive has no index; run ranlib to add one [/workspaces/runtime/src/libraries/System.Buffers/tests/System.Buffers.Tests.csproj]
wasm-ld : error : /workspaces/runtime/artifacts/bin/microsoft.netcore.app.runtime.wasi-wasm/Release/runtimes/wasi-wasm/native/libicui18n.a: archive has no index; run ranlib to add one [/workspaces/runtime/src/libraries/System.Buffers/tests/System.Buffers.Tests.csproj]
wasm-ld : error : /workspaces/runtime/artifacts/bin/microsoft.netcore.app.runtime.wasi-wasm/Release/runtimes/wasi-wasm/native/libicuuc.a: archive has no index; run ranlib to add one [/workspaces/runtime/src/libraries/System.Buffers/tests/System.Buffers.Tests.csproj]
```

Reply:
I'l take a look.

Reply:
https://github.com/dotnet/runtime/issues/81560